src/config.ts

```diff
@@ -63,6 +64,11 @@ class DiscordBridgeConfigLimits {
   public roomGhostJoinDelay: number = 6000;
 }
 
+export class DiscordBridgeConfigPuppeting {
+  public enabled = true;
```

Comment:
Maybe default to `false` for publicly run bridges?

Reply:
Fair.